generators/app/templates/public/locales/es/translation.json

```diff
@@ -142,7 +142,8 @@
         "label": "Saber más",
         "href": "url link"
       }
-    }
+    },
+    "alreadyExists": " archivo o carpeta ya existe en su pod."
```

Comment:
This doesn't have the variable the other languages have {{nameFile}}

Reply:
we don't need to so I removed from the other files.